app/helpers/application_helper.rb

```diff
@@ -66,6 +66,15 @@ def flow_title(flow_object)
     flow_object.title || service.find_page_by_uuid(flow_object.uuid).title
   end
 
+  def fb_back_link(path)
```

Comment:
```suggestion
  def editor_back_link(path)
```
❓

Reply:
Yeah, I toyed with this....

In a lot of places we use `fb` as the prefix still, which is obviously outdated, but continuing to use it adds consistency... so do you change and have differing prefixes across the product or keep it the same and have an old one? 🤷‍♂️

On reflection though I think here your change makes sense, as we don't actually have any fb prefixed helpers, and editor is probably more clear as to where it is used.